singer-io/tap-twitter-ads
# TDL-18006 implement request timeout

# Description of change
- Added an optional `request_timeout` parameter in the config which supports string, int and float values.
- If the `request_timeout` is empty or 0 or not passed in the config, take the default as 300 seconds.

# Manual QA steps
 - Verify that the correct request timeout parameter is passed in the client when provided in config.
 - Verify that it accepts string and float values.
 - Verify that when nothing or 0 is passed, it takes the default value.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
